kernel-laws/jvm/src/test/scala/cats/kernel/laws/JvmLawTests.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2015 Typelevel
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package cats.kernel
+package laws
+
+import cats.kernel.instances.currency.*
+import cats.kernel.laws.discipline.*
+import munit.DisciplineSuite
+import java.util.Currency
+import org.scalacheck.{Arbitrary, Cogen, Gen}
+import scala.jdk.CollectionConverters.*
+
+class JvmLawTests extends TestsConfig with DisciplineSuite {
+  implicit private val arbitraryCurrency: Arbitrary[Currency] = Arbitrary(
+    Gen.oneOf(Currency.getAvailableCurrencies().asScala)
+  )
+  implicit private val cogenCurrency: Cogen[Currency] = Cogen[String].contramap(_.getCurrencyCode())
```

Comment:
Perhaps, it would not hurt if we put these instances into `cats.laws.discipline.arbitrary` to make them available for everyone.

Reply:
I'm wrapping my head around the dependency graph here. Doesn't the `laws` module depend on this one? If we want to add these instances to `kernel-laws`, is there a way to add jvm-only code without moving too many files (`kernel-laws` is defined as `CrossType.Pure`)? Sorry not very used to cross compilation.

Reply:
Yeah, it is quite confusing. If it is not possible to do it in a simple way, let's keep it private in the tests for now then.